docs/3-building-blocks-of-oop-part-2/hometask/src/Consumable.ts

```diff
@@ -1 +1,38 @@
-// your code goes here
\ No newline at end of file
+import { Item } from "./Item";
+
+export class Consumable extends Item {
+  consumed: boolean = false;
+  spoiled: boolean;
```

Comment:
Both these properties should be private according to the UML diagram

Reply:
done